index.html

```diff
@@ -767,18 +850,155 @@ <h2><dfn>CaptureController</dfn></h2>
               </p>
+              <ol>
+                <li>
+                  <p>
+                    Let <var>focusBehavior</var> be the method's first argument.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[IsBound]]}} is <code>false</code>,
+                    [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[Source]]}} is <code>null</code>,
+                    [=exception/throw=] a "{{InvalidStateError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[Source]]}} has been
+                    <a data-cite="GETUSERMEDIA#source-stopped">stopped</a>,
+                    [=exception/throw=] a "{{NotAllowedError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[DisplaySurfaceType]]}} is neither
+                    {{DisplayCaptureSurfaceType/"browser"}} nor {{DisplayCaptureSurfaceType/"window"}},
+                    [=exception/throw=] a "{{NotAllowedError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[FocusWindowClosed]]}} is <code>true</code>,
+                    [=exception/throw=] a "{{NotAllowedError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[InitialFocusLost]]}} is <code>true</code>,
+                    abort these steps.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    The user agent SHOULD abort these steps if too much time has elapsed since
+                    the [=capture-session=] started. The timespan is left up to the user agent,
+                    but it is recommended that a value of one second be used.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Set {{CaptureController/[[FocusWindowClosed]]}} to <code>true</code>.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If <var>focusBehavior</var> is NOT {{CaptureStartFocusBehavior/"focus-captured-surface"}},
+                    abort these steps.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Run the following steps in parallel:
+                  </p>
+                  <ol>
+                    <li>
+                      <p>
+                        Focus the [=display surface=] referred to by {{CaptureController/[[Source]]}}.
```

Comment:
Do we want to mandate something more specific here?

If {{CaptureController/[[DisplaySurfaceType]]}} is `"window"`, do we focus the window? In case that window belongs to the user agent, does it focus the visible document or not? Or do we not want to say?

If {{CaptureController/[[DisplaySurfaceType]]}} is `"browser"`, do we focus the top-level browsing context associated with {{CaptureController/[[Source]]}}'s [=display surface=]?

Reply:
Let's start by leaving it unspecified and letting our respective browsers do what they do today. We can discuss aligning the finer details as a follow-up.